renovate.json

```diff
@@ -19,9 +19,22 @@
     {
       "packageNames": ["google.golang.org/genproto"],
       "schedule": "after 7pm on monday"
+    },
+    {
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "groupName": "all non-major dependencies",
```

Comment:
Renovate creates a branch in its fork and makes all updates for the given package group from that branch.
I tried it once in gapic-generator-typescript and after about a year of successful update, the branch eventually somehow got corrupted and I found no way in Renovate to force it recreate the branch. So if this happens here, we should be ready to rename the group name / slug to force it to switch to a new branch.

Reply:
Thanks for the info! We'll keep in mind.